src/CMakeLists.txt

```diff
@@ -13,14 +13,17 @@ else()
     message(FATAL_ERROR "Magnum context creation is not supported on this platform")
 endif()
 
-set_directory_properties(PROPERTIES CORRADE_USE_PEDANTIC_FLAGS ON)
+
+#set_directory_properties(PROPERTIES CORRADE_USE_PEDANTIC_FLAGS ON)
```

Comment:
Why is this commented out? Was it causing a lot of warnings printed on the output?

In that case, maybe `include_directories(SYSTEM ${GTKMM_INCLUDE_DIRS})` could help.

I tried to find a "modern CMake" way of finding GTK, but sadly there's nothing ready-made and the official guide recommends using pkg-config, which isn't really portable. Oh well.

Reply:
Yeah, that's exactly what was happening, thanks for the tip! It's a shame pkg-config is necessary, I'm hoping they update that when Gtkmm 4 comes out.